types/reapop/index.d.ts

```diff
@@ -0,0 +1,165 @@
+// Type definitions for reapop 1.1
+// Project: https://github.com/LouisBarranqueiro/reapop#readme
+// Definitions by: Barrokgl <https://github.com/Barrokgl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1.1
```

Comment:
The version specified here is not correct. At a minimum to support React, you need to use `// TypeScript Version: 2.3`